routes/target.js

```diff
@@ -0,0 +1,37 @@
+var express = require("express");
+var router = express.Router();
+const db = require("../model/helper");
+
+// Postman Test = OK http://localhost:5000/api/target/test
```

Comment:
Remove these comments as the code is self-explanatory. 

Comments should explain the "why" rather than the "what" when the "what" is self-explanatory from the code.

Reply:
Updated.